tests/specs/contour-utils-spec.js

```diff
@@ -486,6 +498,19 @@ describe('utils', function () {
             it('should pass the correct context to the functions', function () {
                 expect(passCtx).toBe(expectedContext);
             });
-        })
+
+            it('should skip the skip list', function () {
+                expect(res.x).toEqual(input.x);
+                expect(res.d.x).toEqual(input.d.x);
+            });
+
+            it('should skip the skipMatch', function () {
+                expect(res.d.zzz).toEqual(input.d.zzz);
+            });
+
+            it('should skip functions that expect parameters', function () {
+                expect(res.d.other).toEqual(input.d.other);
+            });
```

Comment:
note to self:
Add test to check default call to materialize always includes forces ignore props (ie. /formatter/)